### PR TITLE
Frontend: Foundations for multi tenant frontend

### PR DESCRIPTION
**Pull request by torkelo**
If the hosted grafana proxy that sits in front each instance and proxies all requests could somehow tell Grafana to load a different frontend version (assets) then we could update the frontend independet from the backend (and "host" it just through cdn). 

This PR removes the webpack template plugin in favor of just writing the assets to a json file that the backend can read to know what assets to load. 

We can then later add the ability to read what assets to use from an external api

### CODE REVIEW

File: scripts/webpack/webpack.dev.js [hunk 1]
```diff
@@ -101,26 +101,17 @@ module.exports = (env = {}) => {
...
-        excludeChunks: ['dark', 'light'],
-      }),
-      new HtmlWebpackPlugin({
-        filename: path.resolve(__dirname, '../../public/views/index.html'),
-        template: path.resolve(__dirname, '../../public/views/index-template.html'),
-        inject: false,
-        chunksSortMode: 'none',
-        excludeChunks: ['dark', 'light'],
-      }),
       new HTMLWebpackCSSChunks(),
```
Comment (torkelo) [inline scripts/webpack/webpack.dev.js:104]: @jackw what is this doing? https://github.com/grafana/grafana/pull/47625. 

works without it

File: package.json [hunk 2]
```diff
@@ -329,6 +330,7 @@
     "i18next-browser-languagedetector": "^7.0.2",
     "immer": "10.0.2",
     "immutable": "4.3.1",
+    "install": "^0.13.0",
```
Comment (joshhunt) [inline package.json:333]: did you run `yarn install install`? 😂

Comment (torkelo) [reply package.json]: lol, must have :)

File: pkg/tests/testinfra/testinfra.go [hunk 3]
```diff
@@ -157,6 +157,14 @@ func CreateGrafDir(t *testing.T, opts ...GrafanaOpts) (string, string) {
 	viewsDir := filepath.Join(publicDir, "views")
 	err = fs.CopyRecursive(filepath.Join(rootDir, "public", "views"), viewsDir)
 	require.NoError(t, err)
+	// Copy index template to index.html, since Grafana will try to use the latter
+	err = fs.CopyFile(filepath.Join(rootDir, "public", "views", "index.html"),
```
Comment (torkelo) [inline pkg/tests/testinfra/testinfra.go:161]: @ryantxu is this line ever needed now?

Comment (ryantxu) [reply pkg/tests/testinfra/testinfra.go:161]: it *should* not be -- but looks like some tests fail without it, but i think that is a problem with the tests/setup.  I'll look more soon

**Review by ryantxu — APPROVED** [review]

**Pull request merged**